docs/advanced/transaction-details.md

```diff
@@ -2,11 +2,28 @@
 
 ---
 
-Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. 
+Decred transactions are transfers of DCR that exist within blocks. Transactions are comprised primarily of inputs and outputs, though they have a few other fields of data as well. In this page, we will examine the types of transactions Decred supports, the basic format common to all transactions, and examine an example transaction in detail. 
+
+
+## Transaction Types
+
+To enable Decred's consensus rules, transactions are split into two separate trees within a block: the regular transaction tree and the stake transaction tree. The stake transaction tree contains transactions related to ticket buying and ticket voting rewards. The regular transaction tree contains all other transactions. 
+
+In addition to regular transactions (sending DCR from one address to another), there are a few special transaction types to be aware of:
+
+* **Coinbase transaction:** Every block which is mined contains a single coinbase transaction. This transaction will only have one input, and that is newly created Decred which constitutes the PoW and Treasury portions of the block reward. Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if Proof-of-Stake (PoS) voters vote to reject the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.
```

Comment:
Inputs can't create coins.

Reply:
Changed to 'output',

> Every block which is mined contains a single coinbase transaction. This transaction will only have one **output**, and that is...

Reply:
Unfortunately, that is not true either.  There are many consensus rules around coinbase transactions, but in terms of the number of inputs and outputs, a coinbase transaction is technically only required to have 1 null input, which indicates no existing coins are being spent, and 2 outputs.

The first output creates the coins that are added to the project treasury and is required to be 10% of the total generated block subsidy for that height.  The second output contains the encoded block height which ensures coinbase hash collisions are impossible.

After that, there can be an arbitrary number of additional outputs (up to the maximum limits) so long as the sum of the remaining outputs does not pay more coins than 60% of the total generated block subsidy for that height.

Consequently, a typical coinbase consists of 3 outputs, the required aforementioned 2 outputs and a the third output pays the 60% PoW subsidy portion to the PoW miner.

Reply:
While this is all good info, wary of getting too granular here. Someone really needing this info will probably end up in the source code anyway (e.g. noting that non-typical transactions might have multiple outputs instead of just the output sending 60% to one miner). Also, changing block subsidy to block reward for consistency with other high-level docs. How does this sound?

>Coinbase transaction: Every block which is mined contains a single coinbase transaction. A coinbase transaction will typically have three outputs: one output that contains the encoded block height (used to ensure that coinbase hash collisions are impossible), one output that creates the coins added to the project treasury (10% of the block reward), and one output that creates the coins paid to the PoW miner for creating the block (60% of the block reward). Coinbase transactions are part of the regular transaction tree, which means that they will be rejected if Proof-of-Stake (PoS) voters vote to disapprove the block which contains them. Decred created in coinbase transactions cannot be spent until the coinbase maturity period has passed.

Reply:
Sounds good and is accurate.  Nice job distilling it down to the most relevant bits!

Reply:
Thx! Your writing and feedback is definitely helping me understand the concepts